### PR TITLE
Add label to source PR if backport fails

### PR DESCRIPTION
**Pull request by matthauck**

### CODE REVIEW

File: src/github/api.rs [hunk 1]
```diff
@@ -379,6 +381,20 @@ impl Session for GithubSession {
...
+        #[derive(Serialize)]
+        struct AddLabels {
+            labels: Vec<String>,
+        }
+
+        let body = AddLabels {
+            labels: labels,
+        };
+
+        self.client.post_void(&format!("repos/{}/{}/issues/{}/labels", owner, repo, number), &body)
```
Comment (matthauck) [inline src/github/api.rs:394]: I tested this out locally and looks like github will create a label for you from this API call if the label doesn't exist yet. :tada:

**Review by trevershick — APPROVED** [review]
looks good to me. this will be nice

**Comment by connyay** [comment]
👏

**Review by jmealins — COMMENTED** [review]
LGTM, this will be useful.

**Pull request merged**